src/components/layout/Layout.js

```diff
@@ -9,8 +9,11 @@ const Layout = (props) => {
 
   return (
     <Fragment>
+      <div className={classes.header}>
+
       <MainHeader />
```

Comment:
the indentation doesn't look right
are you working with plugin that auto-indents your code when your make changes?